### ISSUE TITLE
As a developer, I want PUG converter to use single space as initial delimiter for hierachy so that we can save money

### ISSUE THREAD

**Issue by 3wweiweiwu** [3wweiweiwu/recordwright-healing#5]
```
block content
 div#1.row
  div#2 All SERVICES
  ul#3
   li#1 HTML FRONT
   li#2 REACT FRONT
   li#3 CSS DESIGN
   li#4 REACT FRONT
   div#5 OUR WORK
   div#6 FOR AGENCIES
   div#7 ABOUT
   div#8 BLOG
   div#9 CLIENT LOGIN
 div.row#10
  div.column
   div#11 The Kitchen
   div#12 Client Area
```

**Comment by Daniel-V-aspen** [comment]
Implemented in Merge request #14

**Comment by Daniel-V-aspen** [comment]
closed in last comment
